kitchen.cloud.yml

```diff
@@ -0,0 +1,37 @@
+<% require 'kitchen-sync' %>
+---
+driver_config:
+  aws_access_key_id: <%= ENV['AWS_ACCESS_KEY_ID'] %>
+  aws_secret_access_key: <%= ENV['AWS_SECRET_ACCESS_KEY'] %>
+  aws_ssh_key_id: <%= ENV['AWS_KEYPAIR_NAME'] %>
+  availability_zone: <%= ENV['AWS_AVAILABILITY_ZONE'] %>
+
+provisioner:
+  name: chef_zero
+  require_chef_omnibus: latest
+
+platforms:
+  - name: ubuntu-14.04
+    driver_plugin: ec2
+    driver_config:
+      username: ubuntu
+      image_id: <%= ENV['AWS_AMI_UBUNTU_1404_HVM'] %>
+      flavor_id: t2.micro
+    run_list:
+      - recipe[apt]
+
+suites:
+
+  - name: default
+    run_list:
+      - recipe[fastly::default]
+    attributes: {}
+
+  - name: service
+    run_list:
+      - recipe[fastly_test::service]
+    attributes:
+      fastly:
+        api_key: <%= ENV.fetch('FASTLY_API_KEY') %>
+        aws_access_key: <%= ENV.fetch('AWS_ACCESS_KEY_ID') %>
+        aws_secret_key: <%= ENV.fetch('AWS_SECRET_ACCESS_KEY') %>
```

Comment:
Lets move the `api_key` attribute to `kitchen.yml` instead and remove the `aws_*` attributes as they aren't related to this cookbook and were just used with testing on AWS.